receiver/otlpreceiver/config.go

```diff
@@ -22,7 +22,7 @@ const (
 )
 
 type HTTPConfig struct {
-	*confighttp.HTTPServerSettings `mapstructure:",squash"`
+	*confighttp.HTTPServerConfig `mapstructure:",squash"`
```

Comment:
@atoulme this is also a breaking change. Can you please add a changelog entry?

Reply:
Added with new PR here: https://github.com/open-telemetry/opentelemetry-collector/pull/9418